machine_learning/k_means_clust.py

```diff
@@ -47,12 +47,18 @@
             k
         )
 
-  5. Have fun..
+  5. Transfers Dataframe into excel format it must have feature called 
+  'Clust' with clust numbers in it.
```

Comment:
```suggestion
  5. Transfers Dataframe into excel format it must have feature called
      'Clust' with k means clustering numbers in it.
```

Reply:
Travis fails because of the single space character at the end of line 50.  Note that psf/black will autofix this.

Reply:
sorry, i thought i checked it. will do right ahead.